CHANGELOG.md

```diff
@@ -13,8 +13,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Add styling guide documentation for customizing SDK and UI component CSS.
 - Extend and enabled style customizing capabilities on the SDK components.
 ### Changed
-
+- Update meeting manager to use new event controller API.
```

Comment:
Sorry for not catching these at the same time....

Need to update "meeting manager" -> `MeetingManager`, "event controller" -> `EventController`

Reply:
Updated